src/cron/generateInvites.ts

```diff
@@ -0,0 +1,55 @@
+import fastq from 'fastq';
+import { Cron } from './cron';
+import { Feature, FeatureValue, Invite } from '../entity';
+import { generateUUID } from '../ids';
+
+const cron: Cron = {
+  name: 'generate-invites',
+  handler: async (con, logger) => {
+    const featuresQuery = con
+      .createQueryBuilder()
+      .from(Feature, 'f')
+      .where(
+        'f."createdAt" < now() - interval \'7 days\' AND f."value" = :value',
+        { value: FeatureValue.Allow },
+      ).andWhere(`
+        NOT EXISTS (
+          SELECT  1
+          FROM    invite i
+          WHERE   i."userId" = f."userId"
+          AND     i."campaign" = f."feature"
+        )
+      `);
+
+    let count = 0;
+    const queueConcurrency = 1000;
+    const queue = fastq.promise(async ({ feature, userId }: Feature) => {
+      await con.getRepository(Invite).insert({
+        token: generateUUID(),
```

Comment:
There should be a directive for the token property if we end up using uuid. I think it is `@PrimaryGeneratedColumn('uuid')`. This would automatically generate the UUID on DB level and would not require app level generation.

Reply:
@idoshamun @rebelchris do we have any preference on how should the token be generated?

Reply:
@sshanzel Good point, I actually don't remember why we choose to do the `generateMemberToken` function over database level?
Would love to hear @idoshamun opinion would indeed make things easier if we generate on insert.

Reply:
uuid is a standard thing, it doesn't matter where we generate it. So let's go with the easiest solution which is the insert query I guess